modules/masterbar/masterbar.php

```diff
@@ -21,6 +21,7 @@ class A8C_WPCOM_Masterbar {
 	private $primary_site_slug;
 	private $user_text_direction;
 	private $user_site_count;
+	private $tracks_events;
```

Comment:
Can we remove this if it's no longer needed?

Reply:
great catch! Removed on 474d8cef923a9f8f839d733033a496d0a3a28089.